app/src/main/java/explorer/city/com/cityexplorer/Network/TeleportService.kt

```diff
@@ -0,0 +1,41 @@
+package explorer.city.com.cityexplorer.Network
+
+import com.squareup.moshi.Moshi
+import explorer.city.com.cityexplorer.BaseApplication
+import explorer.city.com.cityexplorer.R
+import io.reactivex.schedulers.Schedulers
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import retrofit2.converter.moshi.MoshiConverterFactory
+
+object TeleportService {
```

Comment:
You really shouldn't make things singleton by default. Singleton is a pretty crappy pattern -- you don't know if this needs to be singleton, so you shouldn't make it so.